src/main/java/se/lth/base/server/rest/UserResource.java

```diff
@@ -89,7 +90,9 @@ public User createUser(Credentials credentials) {
         if (!credentials.hasPassword() || !credentials.validPassword()) {
             throw new WebApplicationException("Password too short", Response.Status.BAD_REQUEST);
         }
-        return userDao.addUser(credentials);
+//		TODO: get necessary information from front-end and pass the real info to addUser.
```

Comment:
All information kommer att finnas i credentials objektet, vi pratade ju om att göra om credentialsobjektet till ett Object med ett attribut User i sig

Reply:
Det missade jag, men något jag fixar i framtiden då.

Reply:
Sounds fair, please change the comment so you remember that 😄